coralproject/talk
# closeAsset Mutation

## What does this PR do?

- Adds new `closeAsset` mutation to resolve CI/browsertack time skew issue
- Modifies in-stream configure tab to use the `closeAsset` mutation to close the asset

## How do I test this PR?

1. Visit a comment stream as an admin.
2. Visit the configure tab.
3. Close the comment stream by clicking the "Close Stream" button.
4. Re-visit the comment stream, notice that it is closed!

Related: https://www.pivotaltracker.com/story/show/154406716
